### DISCUSSION TITLE
Custom Reports - Can't access Objectbricks

### DISCUSSION THREAD

**Discussion by CRISTA-AF** [pimcore/pimcore#15835] [Q&A / Support] [answered]
I want to do an sql query from Custom Reports and I need access data that is stored in an objectbrick. 
I try to do this by naming **object_brick_store_(id) Table** in the **FROM** clause of my sql query as insructed in the [Database Model ](https://pimcore.com/docs/pimcore/10.0/Development_Documentation/Development_Tools_and_Details/Database_Model.html )of Pimcore, this however doesn't work

I appreciate your help

**Comment by dvesh3** [comment]
Please share your query and errors that you are facing. It would be easier to support with some background info.

**Comment by CRISTA-AF** [comment]
My custom report query and error are shown here 
<img width="508" alt="CustomReport" src="https://github.com/pimcore/pimcore/assets/142884768/aecb013c-73c8-415c-97ff-8aee515adcd4">

And here is the definition of the objectbrick. I am trying to query for EK and priceInEUR

<img width="598" alt="SaleInformation" src="https://github.com/pimcore/pimcore/assets/142884768/c7fa7d2c-553f-40d4-9899-ab6d5ae9d4a6">

**Reply by dvesh3** [reply, marked as answer]
Please check your database tables. Objectbrick table name is incomplete in your case as the format is `object_brick_store_{BrickName}_{ClassName}` e.g. object_brick_store_SaleInformation_CAR.

Just a side note, you should use query tables for fetching the data in reports like explained here [Database Model](https://pimcore.com/docs/pimcore/10.0/Development_Documentation/Development_Tools_and_Details/Database_Model.html)

**Reply by CRISTA-AF** [reply]
Thank you!!
